website/docs/docs/quickstarts/overview.md

```diff
@@ -0,0 +1,50 @@
+---
+title: About quickstarts
+id: overview
+sidebar_label: "About quickstarts"
+description: "Quickly create your first dbt project with SQL."
+---
+
+Learn how to quickly start using dbt Cloud or dbt Core using a sample project and one of the most common data platforms. The quickstarts will show you how to set up dbt and perform some key tasks. These tasks will illustrate how dbt establishes standard practices for your work such as using Git to version control your analytics code, testing your models to validate they are working correctly, documenting your models in detail to better collaborate with others, and scheduling jobs to execute them at your designated times.  
+
+Before you begin, you will need:
+
+* Warehouse with sample data. If you don't have this, you can use the [BigQuery project](/docs/quickstarts/dbt-cloud/bigquery), which leverages public data sets.
+* Basic understanding of Git.
+* Basic understanding of SQL.
+
+
+<div className="grid--2-col">
+
+<Card
+    title="Quickstart for dbt Cloud and BigQuery"
```

Comment:
Is there a way we can pull the "dbt Cloud is the fastest and most reliable way to deploy dbt." in all of these cards into the intro paragraph?